### PR TITLE
Multiple text fixes for catalog

### PR DESCRIPTION
**Pull request by fxiang1**
Signed-off-by: Feng Xiang <fxiang@redhat.com>

Issues:
- https://github.com/stolostron/backlog/issues/24525
- https://github.com/stolostron/backlog/issues/24649
- https://github.com/stolostron/backlog/issues/24650

Changes:
- Made catalog infrastructure card text more consistent
- Added classic label to standalone card
- Changed the text for Hosted control plane type

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -19,10 +19,18 @@
...
     "A credential stores the access credentials and configuration information for creating clusters.": "A credential stores the access credentials and configuration information for creating clusters.",
+    "A fully capable cluster with a smaller resources requirements and quicker cluster creation.": "A fully capable cluster with a smaller resources requirements and quicker cluster creation.",
```
Comment (matthewcarleton) [inline frontend/public/locales/en/translation.json:22]:
```suggestion
    "A fully capable cluster with a smaller resource requirement and quicker cluster creation.": "A fully capable cluster with a smaller resource requirement and quicker cluster creation.",
```

File: frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateInfrastructure/CreateControlPlane.tsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ export function CreateControlPlane() {
                     {
                         type: CatalogCardItemType.Description,
                         description: t(
-                            'Run OpenShift in a hyperscale manner with many control planes hosted on a central hosting service cluster.'
+                            'A fully capable cluster with a smaller resources requirements and quicker cluster creation.'
```
Comment (matthewcarleton) [inline frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateInfrastructure/CreateControlPlane.tsx:39]:
```suggestion
                            'A fully capable cluster with a smaller resource requirement and quicker cluster creation.'
```

File: frontend/public/locales/en/translation.json (hunk 1 above)
Comment (fxiang1) [reply frontend/public/locales/en/translation.json]: That's a good catch. Thanks Matt!

**Review by jamestalton — APPROVED** [review]
lgtm

File: frontend/public/locales/en/translation.json [hunk 3]
```diff
@@ -19,10 +19,18 @@
...
     "A proxy URL to use for creating HTTP connections outside the cluster. The URL scheme must be HTTP.": "A proxy URL to use for creating HTTP connections outside the cluster. The URL scheme must be HTTP.",
     "A proxy URL to use for creating HTTPS connections outside the cluster. If this is not specified, then httpProxy is used for both HTTP and HTTPS connections.": "A proxy URL to use for creating HTTPS connections outside the cluster. If this is not specified, then httpProxy is used for both HTTP and HTTPS connections.",
+    "A Red Hat OpenShift cluster hosted on the Red Hat OpenStack Platform in your on-premise data center.": "A Red Hat OpenShift cluster hosted on the Red Hat OpenStack Platform in your on-premise data center.",
+    "A Red Hat OpenShift cluster running in a Red Hat Virtualization environment in your on-premise data center.": "A Red Hat OpenShift cluster running in a Red Hat Virtualization environment in your on-premise data center.",
```
Comment (swopebe) [inline frontend/public/locales/en/translation.json:28]: that is hosted

Comment (swopebe) [reply frontend/public/locales/en/translation.json:28]: that is running

File: frontend/public/locales/en/translation.json [hunk 4]
```diff
@@ -19,10 +19,18 @@
...
+    "A Red Hat OpenShift cluster running in a vSphere environment in your on-premise data center.": "A Red Hat OpenShift cluster running in a vSphere environment in your on-premise data center.",
+    "A Red Hat OpenShift cluster running in your AWS subscription.": "A Red Hat OpenShift cluster running in your AWS subscription.",
+    "A Red Hat OpenShift cluster running in your Azure subscription.": "A Red Hat OpenShift cluster running in your Azure subscription.",
+    "A Red Hat OpenShift cluster running in your Google Cloud subscription.": "A Red Hat OpenShift cluster running in your Google Cloud subscription.",
+    "A Red Hat OpenShift cluster running on available hosts from your inventory.": "A Red Hat OpenShift cluster running on available hosts from your inventory.",
```
Comment (swopebe) [inline frontend/public/locales/en/translation.json:33]: all of them "that is running" is the only change I would recommend.

**Pull request merged**